### PR TITLE
Basic Russian Accent

### PR DESCRIPTION
**Pull request by Pangogie**
I want this accent to be applied when wearing a ushanka, but I don't know how to do that.

:cl: Pancake
- add: "Russian accent."

### CODE REVIEW

**Comment by Efruit** [comment]
I can't in good conscience request changes for this, but there should be a second that does actual Latin/Russian transliteration via [ISO/9](https://en.wikipedia.org/wiki/ISO_9) or something.

**Comment by Efruit** [comment]
Also, for posterity's sake, Noto Sans _does_ support Cyrillic out of the box.

File: Content.Server/Speech/EntitySystems/RussianAccentSystem.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        accentedMessage.Replace('K', 'К');
+        accentedMessage.Replace('k', 'к');
+        accentedMessage.Replace('m', 'м');
+        accentedMessage.Replace('h', 'н');
+        accentedMessage.Replace('t', 'т');
+        accentedMessage.Replace('R', 'Я');
+        accentedMessage.Replace('r', 'я');
+        accentedMessage.Replace('Y', 'У');
+        accentedMessage.Replace('W', 'Ш');
+        accentedMessage.Replace('w', 'ш');
```
Comment (moonheart08) [inline Content.Server/Speech/EntitySystems/RussianAccentSystem.cs:34]: This should use a list of replacements, instead of of duplicated code.

**Review by moonheart08 — CHANGES_REQUESTED** [review]

**Comment by mirrorcult** [comment]
this should use the `AddAccentClothing` component recently added in #6194 for ushankas

File: Content.Server/Speech/EntitySystems/RussianAccentSystem.cs (hunk 1 above)
Comment (metalgearsloth) [reply Content.Server/Speech/EntitySystems/RussianAccentSystem.cs:34]: I'm not sure how Replace works internally but a switch statement might be better given the volume?

**Comment by Pangogie** [comment]
Added the `AddAccentClothing`, but I definitely need more guidance on doing the replacements better.

**Pull request merged**